.sanity-ansible-ignore-2.11.txt

```diff
@@ -6,41 +6,53 @@ plugins/module_utils/network_lsr/nm/__init__.py import-3.6!skip
 plugins/module_utils/network_lsr/nm/__init__.py import-3.7!skip
 plugins/module_utils/network_lsr/nm/__init__.py import-3.8!skip
 plugins/module_utils/network_lsr/nm/__init__.py import-3.9!skip
+plugins/module_utils/network_lsr/nm/__init__.py import-3.10!skip
+plugins/module_utils/network_lsr/nm/__init__.py import-3.11!skip
 plugins/module_utils/network_lsr/nm/active_connection.py import-2.6!skip
 plugins/module_utils/network_lsr/nm/active_connection.py import-2.7!skip
 plugins/module_utils/network_lsr/nm/active_connection.py import-3.5!skip
 plugins/module_utils/network_lsr/nm/active_connection.py import-3.6!skip
 plugins/module_utils/network_lsr/nm/active_connection.py import-3.7!skip
 plugins/module_utils/network_lsr/nm/active_connection.py import-3.8!skip
 plugins/module_utils/network_lsr/nm/active_connection.py import-3.9!skip
+plugins/module_utils/network_lsr/nm/active_connection.py import-3.10!skip
+plugins/module_utils/network_lsr/nm/active_connection.py import-3.11!skip
```

Comment:
Maybe this can be dropped, it seems to me that python 3.11 is not supported with ansible 2.11.